.github/workflows/test.yml

```diff
@@ -17,22 +24,20 @@ jobs:
     needs: style
     strategy:
       matrix:
-        python-version: ['3.9', '3.10', '3.11', '3.12']
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
```

Comment:
What about 3.13?

Reply:
I think adding 3.13 can be a separate issue / PR. Maybe there are some numpy version issues we'll have to deal with, and that effort is orthogonal to this effort.